blazar_dashboard/content/leases/templates/leases/_lease_create_network_help.html

```diff
@@ -0,0 +1,14 @@
+{% load i18n %}
+
+{% block help_message %}
+{% if network_enabled %}
+<div class="alert alert-info">
+Network name is required when reserving a network.
+</div>
+{% endif %}
+{% if floating_ip_enabled %}
+<div class="alert alert-info">
+You can just reserve floating IPs.
```

Comment:
I'm not sure this message is necessary, are users confused about what IP reservations are?

Reply:
> This looks awesome, and is very exciting! That lease create UI update is long overdue. How confident are you on this patch? It's a lot of code to review but there's nothing that scary here for me. I think we could try deploying to dev on Monday and then move to prod after that?
> 
> Excellent job!

Thank you @diurnalist ! I only tested this with devstack. I think we'd better test on dev before deploying to prod. I'll do on Tuesday since Monday is a holiday ;)